spec/jobs/account/contacts_export_job_spec.rb

```diff
@@ -89,7 +89,7 @@
       create(:contact, account: account, email: nil, phone_number: nil)
       described_class.perform_now(account.id, user.id, %w[id name email column_not_present], {})
       csv_data = CSV.parse(account.contacts_export.download, headers: true)
-      expect(csv_data.length).to eq(account.contacts.resolved_contacts.count)
+      expect(csv_data.length).to eq(account.contacts.resolved_contacts(use_crm_v2: account.feature_enabled?('crm_v2')).count)
```

Comment:
we don't need this right ? since we have the fallback for the argument. lets only pass this if we are explicitly enabling this flag for an account.